src/sentry/services/hybrid_cloud/rpc.py

```diff
@@ -475,11 +478,14 @@ def dispatch_remote_call(
         charset = response.headers.get_content_charset() or _RPC_CONTENT_CHARSET
         metrics.incr("hybrid_cloud.dispatch_rpc.response_code", tags={"status": response.status})
         response_body = response.read().decode(charset)
-    if response_body:
```

Comment:
Aside from the hook for future metadata, this check is the big reason I wanted to make this change. It's weird that we're representing logical null return values with an empty response body -- not even sure why that happened, but it must be something I did by mistake. The JSON value `null` would be more sensible, and `{"meta": {}, "value": null}` is even better.